interface/src/common/statusBar/statusBar.js

```diff
@@ -0,0 +1,12 @@
+angular.module('statusBar', [])
```

Comment:
Right: I realized we had several places that could benefit from having a "status bar" done in a standardized way, but it's not adage-server specific, so I put it into `interface/src/common`.

The camelCase naming follows the convention throughout ngBoilerplate of using camelCase in JavaScript source and hyphenated-names in HTML. I have done nothing special to make the translation between the two work, so this appears to be a feature of Angular and, I assume, the preferred way to do it. Please correct me if I'm wrong!